newm-server/src/main/kotlin/io/newm/server/features/scheduler/OutletReleaseStatusJob.kt

```diff
@@ -46,6 +51,14 @@ class OutletReleaseStatusJob : Job {
                     outletReleaseRepository.isSongReleased(songId)
                 ) {
                     songRepository.updateSongMintingStatus(songId, MintingStatus.Released)
+                    // When a track reaches the “Released” status, an email communication
+                    // should be triggered and sent to the artist
+                    emailRepository.send(
+                        "",
+                        configRepository.getString(CONFIG_KEY_RELEASED_EMAIL_NOTIFICATION_TITLE),
+                        "messageUrl",
+                        mapOf()
+                    )
```

Comment:
Instead of calling emaiRepository directly here, If you look at updateSongMintingStatus under SongRepositoryImpl, you could see that we already have a placeholder fo the "Released" notification. Look for the comment:
// TODO: Maybe send an email once the song is live on spotify.